packages/css-data/bin/property-value-descriptions.ts

```diff
@@ -72,11 +76,63 @@ const newPropertiesNames = Object.keys(keywordValues)
         typeof propertiesOverrides[property] !== "string")
   );
 
-let retries = 0;
+const newPropertySyntaxes = propertiesAndFunctionsSyntax.filter(
+  (property) =>
+    forceRegenerate ||
+    (typeof propertySyntaxesGenerated[property] !== "string" &&
+      typeof propertySyntaxesGenerated[property] !== "string")
+);
+
+for (let i = 0; i < newPropertySyntaxes.length; ) {
```

Comment:
please use full words whenever possible, e.g. here 
```suggestion
for (let index = 0; index < newPropertySyntaxes.length; ) {
```